spec/acceptance/suites/default/00_simp_server_install_spec.rb

```diff
@@ -53,8 +56,16 @@ def create_r10k_puppetfile(simp_core_puppetfile)
       on(master, 'puppet resource service puppetserver ensure=running')
     end
 
-    it 'should enable trusted_server_facts' do
-      on(master, 'puppet config --section master set trusted_server_facts true')
+    it 'should do some misc configuration' do
+      p_version = on(hosts.first,'puppet --version').stdout.strip
+      if Gem::Version.new(p_version) >= Gem::Version.new('5')
+        on(master, 'puppet config --section master set trusted_server_facts true')
+      end
+
+      # update packages so we can use TLS1.2 to connect to github
+      if master.host_hash[:box] =~ /oel/ and master.host_hash[:platform] =~ /el-6/
+        on(master,'yum upgrade -y git curl openssl nss')
```

Comment:
This should probably go into `simp-beaker-helpers` directly since it's going to be required across the board at this point.

I'm pretty sure there's already a class for taking care of things like this.